src/wp-includes/script-loader.php

```diff
@@ -3519,7 +3519,7 @@ function _wp_theme_json_webfonts_handler() {
 		wp_add_inline_style( 'wp-block-library', $styles );
 	};
 
-	add_action( 'wp_loaded', $fn_register_webfonts );
+	add_action( 'wp_enqueue_scripts', $fn_register_webfonts, 9 );
```

Comment:
While this prevents the error, the error is caused by a call to the database when the tables don't exist (i.e. when WordPress hasn't been installed).

- `script-loader.php` contains 7 uses of `if ( (!) wp_installing() ) {}`.
- Wrapping the `add_action()` in a condition also removes the error:

```php
if ( ! wp_installing() ) {
	add_action( 'wp_loaded', $fn_register_webfonts );
}
```

- Using `wp_installing()` means the hook doesn't need to change.
- Which hook is most appropriate seems like a separate issue to me.
- The only other time this could occur is when the database tables are dropped. In that case, there are bigger concerns than the Webfonts API.

Reply:
I went with an early return in https://github.com/WordPress/wordpress-develop/pull/2638/commits/df56fc3e7c0bf3b7b5dc6b0045b076c42328f565